Espresso.lua

```diff
@@ -1,2 +1,8 @@
 -- Original Espresso
-print("Espresso - 0,80€")
+
+function NameWithPrice(price)
```

Comment:
is this idiomatic lua? I have no effing clue

Reply:
At least regarding all the lua I've seen so far, which definitely isn't much.